bigquery_etl/cli/query.py

```diff
@@ -309,8 +360,10 @@ def schedule(name, sql_dir, project_id, dag, depends_on_past, task_name):
             click.echo(f"Cannot schedule {query_file}. No metadata.yaml found.")
             continue
 
-        if dag:
+        if dag or use_default_dag:
```

Comment:
If both dag name and `--use-default-dag` flags are passed should we error the command to make sure we don't make any assumptions about the user's intentions?

Reply:
This isn't required anymore since the default DAG parameter is not used anymore.